### DISCUSSION TITLE
[Paid Feature] How to config the numbers of colors

### DISCUSSION THREAD

**Discussion by Breedlove-Jason** [izhangzhihao/intellij-rainbow-brackets#2628] [Q&A] [answered]
Your issue may already be reported!
Please search on the [issues](https://github.com/izhangzhihao/intellij-rainbow-brackets/issues) and the [document](https://github.com/izhangzhihao/intellij-rainbow-brackets/blob/2020.3/README.md) and [discussions](https://github.com/izhangzhihao/intellij-rainbow-brackets/discussions) before creating one.

## Please check

- [ ] I already support this project.
- [X ] I already paid for this plugin.
- [ ] I'm on a trial of this plugin.
- [ ] This issue/feature request is not reported before.

## Your programming languages
Javascript using Webstorm

## Current Behavior
* If describing a bug, tell us what happens instead of the expected behavior
The Paid Features are not unlocked, despite an active yearly paid membership. I cannot add the colors for the paid plan bracket levels or any other of the paid features.
![Screenshot_20230319_142736](https://user-images.githubusercontent.com/36088352/226210724-3b679ff2-31c9-4367-9710-a9332f5ee1c6.png)

## Your Environment
Webstorm 2022.3.4
Build #WS-223.8836.50, built on March 15, 2023
Licensed to Jason Breedlove
Subscription is active until May 11, 2023.
Runtime version: 17.0.6+1-b653.34 amd64
VM: OpenJDK 64-Bit Server VM by JetBrains s.r.o

* Plugin version:
2023.2.6
![MyLlinuxEnv](https://user-images.githubusercontent.com/36088352/226211003-fe3cc188-cc71-4f23-a760-fa2b3faf35fe.png)

```
Linux (Pop OS)
Memory: 256MB
Cores: 32

```

**Comment by izhangzhihao** [comment, marked as answer]
hi, you need increase the number of colors to 10 at settings -> rainbow brackets -> number of colors

**Comment by Breedlove-Jason** [comment]
i feel so dumb. thank you very much. i had no idea the window scrolled to the right
